Tasks/Kubernetes/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
     "del": "2.2.0",
-    "js-yaml": "3.6.1"
+    "js-yaml": "3.6.1",
+	"vsts-task-tool-lib": "^0.4.0"
```

Comment:
Remove ^ and fix it to a version 0.4.0. When you add/change a dependency you have to also create a swagger file required for licencing. Create a task to do it if you want to complete the PR today.